Quadratic equation/quadraticequation.py

```diff
@@ -0,0 +1,12 @@
+import cmath  
+a = float(input('Enter a: '))  
+b = float(input('Enter b: '))  
+c = float(input('Enter c: '))  
+  
+# calculating d
+d = (b**2) - (4*a*c)  
+  
+# find two solutions  
+sol1 = (-b-cmath.sqrt(d))/(2*a)  
+sol2 = (-b+cmath.sqrt(d))/(2*a)  
```

Comment:
this formula doesnt cover all the cases. it will throw error when a=0, and d<0. please handle those cases as well.

Reply:
For a = 0, it will become linear equation which can be solved easily. For D<0 take absolute value and add i(iota sqrt(-1)) after second term.